tests/TestCase.php

```diff
@@ -13,6 +13,6 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        Carbon::setTestNow();
+        Carbon::setTestNow(Carbon::now());
```

Comment:
This should still be without the `Carbon::now()` inside - this is meant to reset any carbon mocking between tests

Reply:
reverted